parts/kubernetesmastercustomscript.sh

```diff
@@ -92,6 +93,43 @@ function ensureKubectl() {
     fi
 }
 
+function setNetworkPlugin () {
+    sed -i "s/^KUBELET_NETWORK_PLUGIN=.*/KUBELET_NETWORK_PLUGIN=${1}/" /etc/default/kubelet
+}
+
+function setDockerOpts () {
+    sed -i "s#^DOCKER_OPTS=.*#DOCKER_OPTS=${1}#" /etc/default/kubelet
+}
+
+function configNetworkPolicy() {
+    if [[ ! -z "${APISERVER_PRIVATE_KEY}" ]]; then
+        # on masters
+        ADDONS="calico-configmap.yaml calico-daemonset.yaml"
+        ADDONS_PATH=/etc/kubernetes/addons
+        CALICO_URL="https://github.com/simonswine/calico/raw/master/v2.0/getting-started/kubernetes/installation/hosted/k8s-backend"
+        if [[ "${NETWORK_POLICY}" = "calico" ]]; then
+            # download calico yamls
+            for addon in ${ADDONS}; do
+                curl -sSL --retry 12 --retry-delay 10 "${CALICO_URL}/${addon}" > "${ADDONS_PATH}/${addon}"
```

Comment:
I guess I am not failing in that case. I have just copied that from: https://github.com/Azure/acs-engine/blob/bc344773b5da0d2afe775e6ef1a1e9d21a217726/parts/kubernetesmastercustomdata.yml#L20